config/rbac/role.yaml

```diff
@@ -5,6 +5,42 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
```

Comment:
Do we want this cluster role to have full permissions on everything for secrets? my main thinking is the potential for  updating or patching existing secrets that it didn't create

Reply:
same for certs

Reply:
I think cert-manager needs that but the controller probably just needs read access as it copies the secret into a manifestwork

Reply:
the controller creates the certificate resources

Reply:
oh good point, I just went super-permissive to get everything working and didn't look back. I'll update this now.